sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java

```diff
@@ -105,6 +105,9 @@ public static synchronized void init(
             final BuildInfoProvider buildInfoProvider = new BuildInfoProvider(logger);
             final LoadClass loadClass = new LoadClass();
 
+            // TODO where should this go?
+            options.setDateProvider(new SentryAndroidDateProvider(buildInfoProvider));
```

Comment:
should be fine here? Or put it into `AndroidOptionsInitializer.loadDefaultAndMetadataOptions` ?

Reply:
Sounds good, will move it there.